lib/primitives/address.js

```diff
@@ -225,17 +226,18 @@ class Address extends bio.Struct {
   fromString(data, network) {
     assert(typeof data === 'string');
 
-    const addr = bech32.decode(data);
+    const [hrp, version, hash] = bech32.decode(data);
 
-    Network.fromAddress(addr.hrp, network);
+    Network.fromAddress(hrp, network);
 
-    return this.fromHash(addr.hash, addr.version);
+    return this.fromHash(hash, version);
   }
 
   /**
    * Inject properties from witness.
    * @private
    * @param {Witness} witness
+   * @param {Number} version
```

Comment:
Did you mean to add `version` as an argument to `fromWitness()` ? Or is is the JSDoc comment from an outdated commit?

Reply:
I reverted some changes that I was playing with and accidentally left this in. The intention was to prepare the codebase for new versions, as methods like `fromPubkeyhash` and `fromPubkey` assume version 0. I thought it would be too large of a change for this PR.